### ISSUE TITLE
Can't resolve '@emotion/core' 

### ISSUE THREAD

**Issue by lindaJNlinda** [doczjs/docz#1719]
## Bug Report

**If you're trying to use a package make sure that '@emotion/core' is installed. If you're trying to use a local file make sure that the path is correct.**

docz: 2.4, react, and react-dom 17 in my project, when i run docz dev have the problem.

i have another question，in my package.json just docz react react-dom ,but i find gatsby, gatsby-cli ...... in node_modules

**Comment by sunvictor** [comment]
I encountered the same issue and resolved by downgrading the @emotion/core package.
`npm install @emotion/core@10.1.1`

**Comment by sunvictor** [comment]
> 

I encountered the same issue and resolved by downgrading the @emotion/core package.
`npm install @emotion/core@10.1.1`

**Comment by mortocks** [comment]
@sunvictor 's solution works although obviously not ideal

**Comment by AbdallahAbis** [comment]
Docz should contain its internal packages, I shouldn't install `@emotion/core` manually and have it on my package.json just so docz can use it!

**Comment by dreamline2** [comment]
@AbdallahAbis that's right!! why does not contain self-dependencies, then need to install deps by yourself?? not friendly.

**Comment by HiWayne** [comment]
> Docz should contain its internal packages, I shouldn't install `@emotion/core` manually and have it on my package.json just so docz can use it!


Maybe @emotion has a break-changing in v11, then docz use it. Waiting fix.

**Comment by coalio** [comment]
This is still an issue, maybe some work on this would be appreciated.

**Comment by amusiq** [comment]
> the

it doesn't work for me though

**Comment by Talking-mute** [comment]
> > the
> 
> it doesn't work for me though

me too

**Comment by stefan-nolte** [comment]
Does not work till today

**Comment by Bluestar123** [comment]
still have problems now
